pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -957,8 +957,9 @@ public NamespaceService getNamespaceService() {
         return functionWorkerService;
     }
 
-    public WorkerService getWorkerService() {
-        return functionWorkerService.orElse(null);
+    public WorkerService getWorkerService() throws UnsupportedOperationException {
+        return functionWorkerService.orElseThrow(() -> new UnsupportedOperationException("Pulsar Function Worker is not " +
+                "enabled, probably functionsWorkerEnabled is set to false"));
```

Comment:
I think you can use `getWorkerServiceOpt()` to check if the `workerService` is present? Since the `workerService` might be used in multiple places if we throw an exception here, this means all the places will throw an exception, this is not necessarily the right choice for all operations. So my suggestion is to let the caller to decide the behavior by checking the `getWorkerServiceOpt()`.

Reply:
And please check the failed tests, there are 2 Checkstyle violations

Reply:
Before submitting the code, I forgot to check the code style. I have already checked and resubmitted
thanks

Reply:
Using getWorkerServiceOpt directly may be a more elegant implementation, but it will change too many places, and the expected behavior of most methods is also to throw a clearer exception.
In your opinion, which one is better? 
Simplify the implementation or use a more elegant return value.
Maybe you have a better idea, Thank you again for your advice

Reply:
In my opinion, If throw exception while getWorkerService(), this might break some other places since it might just want to return a null value. As I said before if all the places are ok for handling the exception that getWorkerService() throws, I'm ok. I just worry about introducing some break changes.

Reply:
@codelipenghui I share your opinion.
but if all of the tests are passing, especially integration tests, we should be safe.

Reply:
Ok

Reply:
@codelipenghui Thank you for your suggestion. I understand your consideration.